.stylelintignore

```diff
@@ -0,0 +1 @@
+assets/css/select2.scss
```

Comment:
Any reason to add extra ignorance file, it can be done inside Gruntfile.js too?
```js
stylelint: {
	options: {
		stylelintrc: '.stylelintrc'
	},
	all: [
		'<%= dirs.css %>/*.scss',
		'!<%= dirs.css %>/select2.scss'
	]
}
```